### PR TITLE
Update header/footer, add new SSO 

### PR DESCRIPTION
**Pull request by eadpearce**
https://uktrade.atlassian.net/browse/ED-2410

### CODE REVIEW

File: app/navigator/settings/base.py [hunk 1]
```diff
@@ -92,16 +104,20 @@
 
 AUTH_PASSWORD_VALIDATORS = [
     {
-        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+        'NAME': 'django.contrib.auth.password_validation.'
```
Comment (richtier) [inline app/navigator/settings/base.py]: would be easier to read as:

```
'NAME': (
     'django.contrib.auth.password_validation.'
     'UserAttributeSimilarityValidator'
).
```

File: app/navigator/settings/base.py [hunk 2]
```diff
@@ -182,10 +199,19 @@
...
+HELP_HOST = os.environ.get(
+    'HELP_HOST', 'https://contact-us.export.great.gov.uk/')
 SSO_HOST = os.environ.get('SSO_HOST', 'https://sso.trade.great.gov.uk/')
 PROFILE_HOST = os.environ.get('PROFILE_HOST', 'https://profile.great.gov.uk/')
+SSO_PROXY_LOGIN_URL = os.environ.get(
+    'SSO_PROXY_LOGIN_URL', 'https://sso.trade.great.gov.uk/accounts/login/')
+SSO_PROXY_SIGNUP_URL = os.environ.get(
+    'SSO_PROXY_SIGNUP_URL', 'https://sso.trade.great.gov.uk/accounts/signup/')
+SSO_PROFILE_URL = os.environ.get(
+    'SSO_PROFILE_URL', 'https://profile.great.gov.uk/selling-online-overseas')
```
Comment (richtier) [inline app/navigator/settings/base.py]: feel free to add `directory-constants` to the requirements, and then import the urls as default_urls, like you did elsewhere.

Comment (eadpearce) [reply app/navigator/settings/base.py]: oh yeah. forgot about this. will do

File: app/navigator/settings/base.py (hunk 1 above)
Comment (eadpearce) [reply app/navigator/settings/base.py]: i think this was breaking one of the tests. it works now when i run tests locally

**Pull request merged**